test/integration/corechecks/docker/testdata/exitcode.compose

```diff
@@ -0,0 +1,36 @@
+# This file uses YAML anchors to deduplicate steps
+# see https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
+# and https://learnxinyminutes.com/docs/yaml/
+
+version: '2'
+services:
+  exit0:
+    image: "busybox:latest"
+    entrypoint: /bin/sh
+    command: "-c 'exit 0'"
+    labels:
+        low.card.label: &lowcardlabel "exitlow"
+        high.card.label: &highcardlabel "exithigh"
+    environment:
+        LOW_CARD_ENV: &lowcardenv "exitlowenv"
+        HIGH_CARD_ENV: &highcardenv "exithighenv"
+  exit1:
+    image: "busybox:latest"
+    entrypoint: /bin/sh
+    command: "-c 'exit 1'"
+    labels:
+        low.card.label: *lowcardlabel
```

Comment:
You could go further with deduplication by directly copying the exit0 as template and overriding just the `command`, see https://github.com/docker/compose/issues/1988#issuecomment-153377186 for an example